opml-importer.php

```diff
@@ -102,9 +109,13 @@ function dispatch() {
 
 <h2><?php _e('Importing...', 'opml-importer') ?></h2>
 <?php
-		$cat_id = abs( (int) $_POST['cat_id'] );
-		if ( $cat_id < 1 )
-			$cat_id  = 1;
+		$cat_id = null;
```

Comment:
To maintain the same behaviour as before (minus the potential "undefined index" notice), this should be `1`, not `null`. See: https://3v4l.org/XfCju

This will also affect the rest of the code below which now contains the presumption of `null` having special menaing.

Reply:
For a new site, passing `'link_category' => 1` effectively is the _same_ as "uses default link category". For a pre-existing site for which new information is being added if _may_ be different, so yes, it could be considered a bug fix, but it is also a change in the behaviour of the plugin, which _may_ or _may not_ be desired.

Your PR description doesn't mention anything about changing the behaviour of the plugin, so I reviewed based on it purely being a update for WP compatibility and defensive coding and flagged this change because it doesn't belong in those categories.

I suggest maintaining the same behaviour as before for this PR and submitting a separate PR proposing the behavioural change as that's a different (merge-)decision point than for this PR.